tenant/test/manual_test_cases.md

```diff
@@ -0,0 +1,145 @@
+# **Manual Test Cases**
```

Comment:
I suggest to turn this into tutorial. This type of tests cannot be automated and we should not suggest developer to rely on them.